include/faabric/transport/MessageEndpoint.h

```diff
@@ -86,7 +91,10 @@ class AsyncSendMessageEndpoint final : public MessageEndpoint
                              int portIn,
                              int timeoutMs = DEFAULT_SOCKET_TIMEOUT_MS);
 
-    void send(uint8_t header, const uint8_t* data, size_t dataSize);
+    void send(uint8_t header,
+              const uint8_t* data,
+              size_t dataSize,
+              int sequenceNum = -1);
```

Comment:
This value of `-1` to signify an unset sequence number is used in a few places as a magic number. It would be good to define it as a constant.